resources/views/individuals/partials/about.blade.php

```diff
@@ -6,9 +6,15 @@
     {{ __('Languages :name uses', ['name' => $individual->first_name]) }}</h3>
 
 <ul>
-    @foreach ($individual->working_languages as $language)
-        <li>{{ get_language_exonym($language) }}</li>
-    @endforeach
+    @if ($individual->working_languages)
+        @foreach ($individual->working_languages as $language)
+            <li>{{ get_language_exonym($language) }}</li>
+        @endforeach
+    @else
+        @foreach ($individual->languages as $language)
```

Comment:
These are actually the languages into which the individual can translate their page. We actually want to retrieve the users's `locale` and (if set) `signed_language` attribute.

Reply:
See: https://github.com/accessibility-exchange/platform/blob/3d7a134f0c7b9f6681ee53ed1568743e80c63dbf/app/Http/Controllers/IndividualController.php#L115-L121